### ISSUE TITLE
comments not supported in snoop.conf

### ISSUE THREAD

**Issue by thomasrebele** [HBPMedical/PostgresRAW#3]
Comments like shown in the readme make queries on table persons return empty result
```
filename-1 = '/home/NoDB/datafiles/load.txt'  # Link to data file
relation-1 = 'persons'                        # Table name (dummy table in the database)
delimiter-1 = ','                             # Delimiter for the file
```
However, if I remove the comments it works:
```
filename-1 = '/home/NoDB/datafiles/load.txt'
relation-1 = 'persons'
delimiter-1 = ','
```

**Comment by amolma** [comment]
The current version allows only full line comments starting line with '#'.

e.g., 
```
# Link to data file
filename-1 = '/home/NoDB/datafiles/load.txt'
# Table name (dummy table in the database)
relation-1 = 'persons'
# Delimiter for the file     
delimiter-1 = ',' 
```

**Comment by thomasrebele** [comment]
thanks for this information! The snippet of the snoop.conf in the Readme.md is somewhat misleading. Could you change it there, so that other people don't make the same mistake?

**Comment by sambuc** [comment]
Thank you for you report. As suggested I fixed the documentation with 3aae82b0069e028395bd60c55aeb56eeef0195c2.

Kind regards

**Comment by thomasrebele** [comment]
thank you very much!
